generated/nidaqmx/constants.py

```diff
@@ -80,8 +75,8 @@ class ActiveOrInactiveEdgeSelection(Enum):
 class AngleUnits(Enum):
     DEGREES = 10146  #: Degrees.
     RADIANS = 10273  #: Radians.
-    TICKS = 10304  #: Ticks.
     FROM_CUSTOM_SCALE = 10065  #: Units a custom scale specifies. If you select this value, you must specify a custom scale name.
+    TICKS = 10304  #: Ticks.
```

Comment:
FYI: Python 3 enums remember the order in which values are defined and allow iterating over the values in definition order at run time.
https://docs.python.org/3/library/enum.html#enum.EnumType.__iter__

However, Python 2 enums didn't remember definition order by default. You had to set the `_order_` attribute. So these enums were not always ordered.

Changing the enum value order breaks compatibility, but very few DAQmx applications are likely to depend on this, and upgrading from Python 2 to Python 3 already changed this behavior.

Reply:
Yes, don't do it here. It would make reviewing the changes a nightmare